src/api/app/models/issue.rb

```diff
@@ -10,6 +10,8 @@ class NotFoundError < APIError
   belongs_to :issue_tracker
   belongs_to :owner, class_name: 'User'
 
+  validates :name, format: { with: /\A[\d\-]+\z/, message: 'only digits and hyphens' }
```

Comment:
This is wrong. `CVE-2018-2842` is a valid issue name (please make sure to extend the model test for this)

Reply:
Sorry about that! I simply copied the changes from PR#4128. I'm quite new to rspec and have tried my best to write a model test. Please suggest any improvements.